consensus/tests/engine_v2_tests/adaptor_test.go

```diff
@@ -67,7 +67,7 @@ func TestAdaptorGetMasternodesFromCheckpointHeader(t *testing.T) {
 	masternodesV1 := adaptor.GetMasternodesFromCheckpointHeader(headerV1)
 	headerV2 := currentBlock.Header()
 	headerV2.Number.Add(blockchain.Config().XDPoS.V2.SwitchBlock, big.NewInt(1))
-	headerV2.Validators.CurrentEpoch = common.Hex2Bytes("0278c350152e15fa6ffc712a5a73d704ce73e2e103d9e17ae3ff2c6712e44e25b09ac5ee91f6c9ff065551f0dcac6f00cae11192d462db709be3758c")
+	headerV2.Validators.CurrentEpoch = []common.Address{}
```

Comment:
do we need "0278c350152e15fa6ffc712a5a73d704ce73e2e103d9e17ae3ff2c6712e44e25b09ac5ee91f6c9ff065551f0dcac6f00cae11192d462db709be3758c" split into address?

Reply:
yes we need, but I don't know what exactly addresses are. Just split into 3?
```
0278c350152e15fa6ffc712a5a73d704ce73e2e1
03d9e17ae3ff2c6712e44e25b09ac5ee91f6c9ff
065551f0dcac6f00cae11192d462db709be3758c
```

Reply:
yes

> yes we need, but I don't know what exactly addresses are. Just split into 3?
> 
> ```
> 0278c350152e15fa6ffc712a5a73d704ce73e2e1
> 03d9e17ae3ff2c6712e44e25b09ac5ee91f6c9ff
> 065551f0dcac6f00cae11192d462db709be3758c
> ```